docker/nodesource_setup_14.x.sh

```diff
@@ -0,0 +1,361 @@
+#!/bin/bash
```

Comment:
Looks like this is failing in the build: 

```
sudo sh docker/build_cpu.sh
The command '/bin/sh -c sh node_setup.sh' returned a non-zero code: 1
```

Reply:
How do I see these build results? I can't see to repro this locally

Reply:
clone this repo and run `sudo sh docker/build_cpu.sh`

Reply:
Hmm, I've deleted the repo, all related docker images and re-cloned, ran with `--no-cache` and can't seem to get this error.

Looks like the cloudbuild also works: https://pantheon.corp.google.com/cloud-build/builds;region=global/d0265607-4b96-4260-b702-30589f45f7ee?project=kaggle-cicd

Possibly it was a transient one-off error? (it still has to download binaries from the internet, so that could always fail). Would you mind trying again? And if you still get it, can you share the build log?

Reply:
I'll run it again

Reply:
Hrm, still failing for me, but I see that it is passing cloud build. Must be an issue on my machine.